examples/siro_sandbox/sandbox_app.py

```diff
@@ -70,6 +77,17 @@ def get_pretty_object_name_from_handle(obj_handle_str):
 @requires_habitat_sim_with_bullet
 class SandboxDriver(GuiAppDriver):
     def __init__(self, args, config, gui_input):
+        self._dataset = config.habitat.dataset
+        self._num_recorded_episodes = 0
+
+        if args.save_filepath_base:
+            self._step_recorder = StepRecorder()
+            self._save_filepath_base = args.save_filepath_base
+            self._episode_recorder_dict = {}
+        else:
+            self._step_recorder = NullRecorder()
```

Comment:
The NullRecorder makes it easier to write recording logic in controllers.py (don't have to check for null recorder).